### PR TITLE
Convert "login" link to "Sign in with Github" button

### PR DESCRIPTION
**Pull request by Khanjan04**
Convert "login" link to "Sign in with Github" button.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -32,6 +32,19 @@ nav .btn-login {
   float: right;
 }
 
+nav .button{
+  font-size: 20px;
```
Comment (swarajpure) [inline css/main.css:36]: Doesn't the font size look too huge compared to rest of the page? Please try reducing the size and see which one looks better. After changing the font-size you'd also have to change the padding to compensate.
image

File: css/main.css [hunk 2]
```diff
@@ -32,6 +32,19 @@ nav .btn-login {
...
+  border-radius: 06px;
```
Comment (swarajpure) [inline css/main.css:37]: `6px` would be more readable than `06px`

File: index.html [hunk 3]
```diff
@@ -36,7 +36,7 @@
           class="btn-login"
           href='https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97'
         >
-          Login
+        <button class="button">Sign In With Github <img class="icon" src="img/github.png" alt="Github Icon" /></button>
```
Comment (swarajpure) [inline index.html:39]: Wouldn't something like `button class="login-btn-txt"` make more sense?
The class name should be self explanatory about it's functioning, right?

Same for img as well. `img class="logo"` would be more self explanatory.

**Review by swarajpure — CHANGES_REQUESTED** [review]
Thank you for this pull request! 🎉 
Requesting some minor changes. Please see to it.

File: css/main.css (hunk 1 above)
Comment (Khanjan04) [reply css/main.css:36]: Okay let me do that

File: css/main.css (hunk 2 above)
Comment (Khanjan04) [reply css/main.css:37]: ok

File: index.html (hunk 3 above)
Comment (Khanjan04) [reply index.html:39]: Ok. Let me change that

Comment (Khanjan04) [reply index.html]: Have done the changes. Please check them.

**Review by swarajpure — APPROVED** [review]
Great job! Thank you! 🎉 😄

**Pull request merged**